src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,14 @@
 package mate.academy
 
+import mate.academy.figure.Figure
+import mate.academy.service.FigureSupplier
+
 fun main() {
-    // draw all figures here
+    val figures: Array<Figure> = arrayOf(
```

Comment:
Please try to find more flexible and scalable solution here.
For example,  what if there will be 100 figures ?